code/game/objects/effects/landmarks.dm

```diff
@@ -574,6 +574,51 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/awaystart) //Without this away mission
 	new /obj/item/clothing/under/rank/civilian/mime/sexy(src.loc)
 	qdel(src)
 
+/// Mob spawners
+/obj/effect/landmark/mob/goliath
```

Comment:
early review but `landmark/mob` is kinda generic and `mob` is already a keyword. Maybe it could be `mob_spawner`?

Reply:
addressed